src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a contact
+ */
+public class Contact {
+    public String value;
+    public boolean isPrivate;
```

Comment:
Should we be exposing our variables here with `public` scope? Perhaps we can look at using the `protected` scope to achieve the same intended functionality?